### ISSUE TITLE
Add a way to expose which extensions are useful to my app

### ISSUE THREAD

**Issue by hadess** [flatpak/flatpak#123]
The gst-libav runtime extension I made in https://github.com/hadess/flatpak-fusion extends the fd.o platform, but it's really an extension for my application, a video player.

How do I tell gnome-software "this type of extension is useful to me" without knowing what those extensions would be, or how they would be made available.

**Comment by alexlarsson** [comment]
You can make it an application extension, but that limits the use of the extension to your app. Maybe we should just add a field to the metadata to claim interest in a particular runtime extension.

**Comment by hadess** [comment]
> Maybe we should just add a field to the metadata to claim interest in a particular runtime extension.

That would suit me, and would allow gnome-software to show related extensions for that application.

**Comment by Mikenux** [comment]
@alexlarsson @hadess

Has this been resolved? On https://github.com/hadess/flatpak-fusion:

> gst-libav is included as a Freedesktop.org runtime extension that's automatically installed

**Comment by hadess** [comment]
Yeah, the `org.freedesktop.Platform.ffmpeg-full` extension now gets automatically installed as needed for runtime extensions, and application extensions can also get automatically installed.
